src/Sarif.Converters/PyLintObjectModel/PyLintLogReader.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.IO;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace Microsoft.CodeAnalysis.Sarif.Converters.PyLintObjectModel
+{
+    public static class PyLintLogReader
```

Comment:
That looks like it would do the job!

---
In reply to: [148077081](https://github.com/Microsoft/sarif-sdk/pull/661#discussion_r148077081) [](ancestors = 148077081)